expyfun/analyze/_analyze.py

```diff
@@ -216,17 +256,18 @@ def wrapper(*args):
     return namedtuple('params', p_types)(**kwargs)
 
 
-def rt_chisq(x, axis=None):
+def rt_chisq(x, axis=None, warn=True):
     """Chi square fit for reaction times (a better summary statistic than mean)
 
     Parameters
     ----------
     x : array-like
         Reaction time data to fit.
-
     axis : int | None
         The axis along which to calculate the chi-square fit. If none, ``x``
         will be flattened before fitting.
+    warn : bool
+        If True, warn about possible bad reaction times.
```

Comment:
thanks for this.  The criterion for warning is too stringent, and always blows up my console.